news/_posts/2022/03/04/contributor-interview-jean-francois-viguier.md

```diff
@@ -0,0 +1,96 @@
+
+The PrestaShop company supports the association and PrestaShop employees regularly intervene in our roundtables.
+
+**How did it start?**
+
+The association was created during the first lockdown in France. At that time, only the Slack channel existed. 
+
+Because of the Covid-19, many French merchants were in very difficult situations, due to sanitary restrictions and the closure of their physical stores. So, I suggested the FOP community to launch a solidarity initiative. Nearly 50 volunteers mobilized to create free basic ecommerce websites for hundreds of companies to digitize and survive the crisis. It was very successful. We had the support of several regions and the city of Paris. And because the initiative had grown so much, we needed to establish a legal framework. So we created the Friends of Presta association!
+
+**Do you have any advice for first-time contributors?**
+
+Sometimes, the process can seem a bit complicated. Don’t worry. Focus on the content of your pull request, there will always be someone to help you with the rest.
+At Friends of Presta, we try to help first-time contributors. We have a dedicated channel for contribution on Slack. By the way, contributing to the FOP console first can be a good idea. The level of requirement is lower there. You’ll then be able to tackle the official PrestaShop repository when you feel ready.
+
+**As a friend of PrestaShop, how do you expect to see the project evolve in 2022?**
+
+V8 is a more “technical” version. For the next versions, I think we should focus on 2 main issues: the cart rules and the checkout (process, logic, calculation). I am aware that these subjects are a bit like a house of cards. If you change something, everything else could be affected. But I think these are two essential topics for merchants.
+
+I also think that we should simplify the contribution process. I sometimes see motivated people on Twitter who want to contribute. But some of them struggle with the process. We need to simplify things and provide more support.
+
+Finally, I think that the differentiation between the PrestaShop company and the open source project is a great thing. I hope that it will encourage the contribution of bigger agencies. It's unfortunate that not so many developers contribute.
+
+**Would you like to add anything?**
+
+Yes! We often hear that the 1.6 is better than the 1.7. This is an urban legend! Many developers were disturbed by this change, but there are really no blocking points in 1.7.
+
+Now that I've had my little rant (:innocent:), I’d like to thank some people.
+
+The first person I’d like to thank is Valentin Szczupak. He and the community have done a lot of work on the new theme. I really hope it will be available in the next version. 
```

Comment:
😘  @NeOMakinG

Reply:
Can we replace it with my nickname? I know my personal name isn't that hard to find, but at least it makes it a bit harder...

Reply:
At least it's a better solution because peoples mainly know me as `NeOMakinG`, but then why would we even need my personal name?

Reply:
I guess it's because Jean Francois did say "Valentin" and not "Ne0MakinG" ;) this interview refers to what he said

Reply:
We could write "Valentin" and not your last name?